src/metabase/integrations/slack.clj

```diff
@@ -137,6 +142,8 @@
     (when invalid-token?
       (log/warn (u/colorize :red (str "🔒 Your Slack authorization token is invalid or has been revoked. Please"
                                       " update your integration in Admin Settings -> Slack."))))
+    (when missing-channel?
+      (log/warn (u/colorize :red "🔍 The Slack channel you've configured doesn't exist. Please update it.")))
```

Comment:
I tried to log the channel name here as well but it's lost somewhere higher in the calls. Do you think I should pull that in as well?

Reply:
I don't think we need this log tbh. we have the log for invalid-token because it's instance wide error. this missing channel is per-notification and we'll log this in task history.

Reply:
Agreed. I'll remove it.